httpclients/httpclient/src/main/java/com/okta/sdk/impl/http/httpclient/HttpClientRequestExecutor.java

```diff
@@ -336,4 +305,18 @@ private HttpHeaders getHeaders(HttpResponse response) {
 
         return headers;
     }
+
+    private static int parseConfigValue(String key, int defaultValue, String warning) {
+
+        int configuredValue = defaultValue;
+        String configuredValueString = System.getProperty(key);
+        if (configuredValueString != null) {
```

Comment:
should this `if` be null or empty?

Reply:
`null`, as it isn't valid key and should stay unset